### ISSUE TITLE
Failure to create a new wallet when a previous wallet created and password is forgotten 

### ISSUE THREAD

**Issue by Devinml** [joinmarket-webui/jam#774]
**Expected behavior**
I should be able to create multiple wallets if I forget the password of a wallet. 

**Actual behavior**
When trying to create a new wallet when the old wallet password was forgotten it routes you to a page that says you must lock your wallet. 
<img width="963" alt="image" src="https://github.com/joinmarket-webui/jam/assets/9991351/d9ae9ba2-b0e4-4ff8-a692-fa2d86037d2e">
 When "Create new wallet" is pressed I get this screen: Only problem is I don't have the old password to lock the old "test wallet"

<img width="963" alt="image" src="https://github.com/joinmarket-webui/jam/assets/9991351/a359d486-a6e7-44b1-ad1f-3f67732f4c4b">

**Steps to reproduce the problem**
  1. Create a wallet
  2. Leave it unlocked and try to create a new wallet. 
  

**Specifications**
  - Version: v0.2.0
  - Platform: umbrel
  - Browser: brave/chrome

**Comment by theborakompanioni** [comment]
Hey @Devinml! 

Thanks for the report. :pray: 

 Yes, indeed, that is a problem. However, the password is needed in order to "lock" an active wallet and "unlock" or create/import another one. A possible workaround is to restart the service (e.g. with a system reboot). I do not think there is anything that can be done otherwise (but I might be wrong!).

**Comment by editwentyone** [comment]
but lets add this piece of information also: 

"… if you have forgot your password, restart the service so that this wallet gets locked and you can proceed."

(bad wording I know)

**Comment by theborakompanioni** [comment]
> but lets add this piece of information also:
> 
> "… if you have forgot your password, restart the service so that this wallet gets locked and you can proceed."
> 
> (bad wording I know)

Aww man.. Do you really think that is necessary?
Let's wait at least for an additional report of such a situation before we clutter the UI for such niche situations. Or am I on the wrong track here?

**Comment by theborakompanioni** [comment]
What I would do is renaming the `active` label from "Unlocked" back to what we had earlier: "Active".
(It is strange to unlock something that is already unlocked, is it not?)

**Comment by Devinml** [comment]
As a user I found it confusing, I think there should be an active and then an unlocked.

**Comment by theborakompanioni** [comment]
> As a user I found it confusing, I think there should be an active and then an unlocked.

Yeah, but that would not really resolve the problem at hand. You need a valid session token to lock or unlock the active wallet in order to create/import a new one. There is only so much you can do, if a user forgets the password. I am afraid, there is nothing we can do about it technically, but at least try to reduce the confusion.

**Comment by Devinml** [comment]
What about adding this to an FAQ section to the readme. I'd be happy to submit that as a pr.

**Comment by theborakompanioni** [comment]
> What about adding this to an FAQ section to the readme. I'd be happy to submit that as a pr.

Hey @Devinml, really appreciate your thinking and willingness to help. Thanks brother.
Since this is the first time someone asked, it is not really a "Frequently Asked Question".

Okay for you to wait and see whether other people have a similar issue with this? Thanks again :pray:
